pom.xml

```diff
@@ -1176,5 +1189,19 @@ under the License.</licenseText>
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>pmd</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-pmd-plugin</artifactId>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
```

Comment:
Is this profile still required given that the plugin is no enabled by default (above lines 828..)?

Reply:
it was requested to do so, but its not working. The activation, which i like, should be removed. @gnodet